src/ppx_deriving.ml

```diff
@@ -2,6 +2,7 @@ open Longident
 open Location
 open Asttypes
 open Parsetree
+open Ast_mapper
```

Comment:
Oh. Maybe just add Ast_mapper.{} around that? I would really prefer not to have get_cookie and such in global scope, it's dirty enough anyway.